.coveragerc

```diff
@@ -281,6 +281,10 @@ omit =
     homeassistant/components/flunearyou/__init__.py
     homeassistant/components/flunearyou/sensor.py
     homeassistant/components/flux_led/light.py
+    homeassistant/components/fmi/__init__.py
+    homeassistant/components/fmi/const.py
+    homeassistant/components/fmi/weather.py
+    homeassistant/components/fmi/config_flow.py
```

Comment:
The configuration flow should not be excluded from the test coverage (besides, I saw there was a test for it 😄 )